ecosystem/resources/members/qmuvi.toml

```
@@ -0,0 +1,15 @@
+name = "qmuvi"
+url = "https://github.com/garymooney/qmuvi"
+description = "A library converting quantum circuits into audiovisual experiences, bridging the gap between complex quantum computations and human perception. It renders music videos that reveal the evolution of quantum states during algorithm processing, making quantum computing more intuitive and accessible."
+licence = "GNU Library or \"Lesser\" General Public License (LGPL)"
+contact_info = "gary.mooney4444@gmail.com"
+alternatives = "_No response_"
+affiliations = "The University of Melbourne"
+labels = [ "Converter", "Visualization",]
+tier = "Community"
+website = "https://garymooney.github.io/qmuvi/"
```

```suggestion
documentation = "https://garymooney.github.io/qmuvi/"
```
We don't have an option for this in the issue form yet